scripts/Constants.s.sol

```diff
@@ -7,18 +7,8 @@ import "contracts/utils/Constants.sol";
                                                    MAINNET CONSTANTS                                                
 //////////////////////////////////////////////////////////////////////////////////////////////////////////////////*/
 
-address constant ACCESS_CONTROL_MANAGER = 0x5bc6BEf80DA563EBf6Df6D6913513fa9A7ec89BE;
-address constant CHAIN_AGEUR = 0x1a7e4e63778B4f12a199C062f3eFdD288afCBce8;
-address constant GOVERNOR = 0xdC4e6DFe07EFCa50a197DF15D9200883eF4Eb1c8;
-address constant GUARDIAN = 0x0C2553e4B9dFA9f83b1A6D3EAB96c4bAaB42d430;
-address constant PROXY_ADMIN = 0x1D941EF0D3Bba4ad67DBfBCeE5262F4CEE53A32b;
-address constant PROXY_ADMIN_GUARDIAN = 0xD9F1A8e00b0EEbeDddd9aFEaB55019D55fcec017;
-address constant TREASURY_EUR = 0x8667DBEBf68B0BFa6Db54f550f41Be16c4067d60;
-address constant TREASURY_USD = 0xf8588520E760BB0b3bDD62Ecb25186A28b0830ee;
 address constant IMMUTABLE_CREATE2_FACTORY_ADDRESS = 0x0000000000FFe8B47B3e2130213B802212439497;
```

Comment:
Are we sure this is the address on every chain?

Reply:
the immutable factory is the same on every chain